common/test/acceptance/pages/studio/users.py

```diff
@@ -64,7 +64,7 @@ def click_add_button(self):
         """
         Click on the "New Team Member" button
         """
-        self.q(css='.create-user-button').click()
+        self.q(css='.create-user-button').first.click()
```

Comment:
What happens when you click this button? Is there an ajax call? If so then there should be a wait_for_ajax after it. Or if it causes a new element to appear / change class / etc. then there should be a waiter for that. 
Trying to anticipate future synchronization issues...

May be outside the scope of this PR but looking at this class, the delete_user_from_course and add_user_to_course should also be improved so that after you take an action you wait for whatever is supposed to happen.